src/SigSpec.CodeGeneration.CSharp.Tests/GeneratorTests.cs

```diff
@@ -28,15 +31,14 @@ public GeneratorTests()
 		[InlineData(typeof(HubWithString), "string")]
 		[InlineData(typeof(HubWithStringOptional), "string")]
 		[InlineData(typeof(HubWithInt), "int")]
-		[InlineData(typeof(HubWithIntOptional), "int")]
+		[InlineData(typeof(HubWithIntOptional), "int?")]
 		[InlineData(typeof(HubWithGuid), "System.Guid")]
-		[InlineData(typeof(HubWithGuidOptional), "System.Guid")]
+		[InlineData(typeof(HubWithGuidOptional), "System.Guid?")]
 		[InlineData(typeof(HubWithStruct), "TestStruct")]
 		[InlineData(typeof(HubWithNullableStruct), "TestStruct")]
 		[InlineData(typeof(HubWithStructOptional), "TestStruct")]
```

Comment:
I believe HubWithNullableStruct and HubWithStructOptional should both be `?` or `Nullable<T>` here.